test/query.test.js

```diff
@@ -2357,77 +2357,71 @@ describe('query', () => {
   test('arity', () => {
     // By default assume all functions should have strict arity
     var testParams = {
-      Ref: [3, 'from 1 to 2'],
-      Do: [0, 'at least 1'],
-      Lambda: [3, 'from 1 to 2'],
-      Call: [0, 'at least 1'],
-      Get: [3, 'from 1 to 2'],
-      Paginate: [3, 'from 1 to 2'],
-      Exists: [3, 'from 1 to 2'],
-      Create: [3, 'from 1 to 2'],
-      Match: [0, 'at least 1'],
-      Union: [0, 'at least 1'],
-      Merge: [0, 'from 2 to 3'],
-      Intersection: [0, 'at least 1'],
-      Difference: [0, 'at least 1'],
-      Format: [0, 'at least 1'],
-      Concat: [0, 'at least 1'],
-      Casefold: [0, 'at least 1'],
-      FindStr: [0, 'from 2 to 3'],
-      FindStrRegex: [0, 'from 2 to 4'],
-      NGram: [0, 'from 1 to 3'],
-      Repeat: [0, 'from 1 to 2'],
-      ReplaceStrRegex: [0, 'from 3 to 4'],
-      SubString: [0, 'from 1 to 3'],
-      Equals: [0, 'at least 1'],
-      Select: [4, 'from 2 to 3'],
-      Add: [0, 'at least 1'],
-      BitAnd: [0, 'at least 1'],
-      BitOr: [0, 'at least 1'],
-      BitXor: [0, 'at least 1'],
-      Divide: [0, 'at least 1'],
-      Max: [0, 'at least 1'],
-      Min: [0, 'at least 1'],
-      Modulo: [0, 'at least 1'],
-      Multiply: [0, 'at least 1'],
-      Round: [0, 'at least 1'],
-      Subtract: [0, 'at least 1'],
-      Trunc: [0, 'at least 1'],
-      Hypot: [0, 'at least 1'],
-      Pow: [0, 'at least 1'],
-      LT: [0, 'at least 1'],
-      LTE: [0, 'at least 1'],
-      GT: [0, 'at least 1'],
-      GTE: [0, 'at least 1'],
-      And: [0, 'at least 1'],
-      Or: [0, 'at least 1'],
-      Index: [3, 'from 1 to 2'],
-      Class: [3, 'from 1 to 2'],
-      Collection: [3, 'from 1 to 2'],
-      Database: [3, 'from 1 to 2'],
-      Function: [3, 'from 1 to 2'],
-      Role: [3, 'from 1 to 2'],
-      Classes: [2, 'up to 1'],
-      Collections: [2, 'up to 1'],
-      Databases: [2, 'up to 1'],
-      Indexes: [2, 'up to 1'],
-      Functions: [2, 'up to 1'],
-      Roles: [2, 'up to 1'],
-      Keys: [2, 'up to 1'],
-      Tokens: [2, 'up to 1'],
-      Credentials: [2, 'up to 1'],
+      Ref: [3, 'from 1 to 2', 'Ref'],
```

Comment:
@BrunoQuaresma @fauna-rodrigo I updated this test to reflect some of the changes I've made. However, the assertions we previously had did not really test all that much.

If y'all have any suggestions regarding what you would like our tests to cover here, I'm open to suggestions! 🙏

Reply:
Maybe we could add an extra assertion to see if the documentation link is included. What do you think?

Reply:
@BrunoQuaresma Sounds good to me! I'll add that and an assertion to confirm that the function name is in the first line of the log as well 👍